meerk40t/core/cutcode.py

```diff
@@ -219,15 +220,15 @@ def contains_uncut_objects(self):
             return False
         for c in self.contains:
             for pp in c.flat():
-                if pp.permitted:
+                if pp.permitted and pp.burns_remaining >= 1:
```

Comment:
pp.permitted should probably be trashed.

Reply:
See my comment above about `inner_selection_cutcode` - if I understood what it was doing, then I can probably remove `permitted` altogether, but I have learned the hard way not to touch stuff unless I know for sure what it is doing.